test_data/yast/autoyast_reinstall/autoyast_reinstall_64bit.yaml

```diff
@@ -10,6 +10,23 @@ file_system:
   /: btrfs
 swap: 1
 profile:
+  suse_register:
+    addons:
+      - addon:
+          unique_key: name
+          name: 'sle-module-server-applications'
```

Comment:
those quotes are not needed for any of the fields in all files. I think only apply when we want to escape something in particular.

Reply:
I thought to use quotes for consistency with the variables. You have mentioned before that it was decided to use quotes for var values like MULTIPATH: '1'

Reply:
I noticed that the rest of test data don't have quotes so, it's better to be consistent with that. Quotes removed. I will run vr for one machine again.

Reply:
In case of variables like MULTIPATH, we have to put quotes, because values are read as strings. But this is valid for Job Group settings, for yaml we can skip wrapping numeric values in quotes.